wagtail/utils/setup.py

```diff
@@ -35,6 +35,7 @@ def bump_client_version(self):
             package = json.loads(input_file.read().decode("utf-8"))
         except ValueError as e:
             print("Unable to read " + path + " " + e)  # noqa: T201
+            input_file.close()
```

Comment:
Suggestion: Rather than manually closing the file, a context manager would work better here.

Issue: In this case, the file is only closed on error, not in the successful case. A context manager would handle that. 

I'm also not sure how much it matters about closing file handles if the program is about to exit, but I agree it's nicer to do things _properly_.

Reply:
Thanks for such detailed reply. I've pushed another commit that'll hopefully address both issues. Please let me know if you'd like me to squash the changes into a single commit or not.